hera_cal/utils.py

```diff
@@ -3,10 +3,12 @@
 import astropy.constants as const
 from astropy.time import Time
 import pyuvdata.utils as uvutils
-from pyuvdata import UVCal
+from pyuvdata import UVCal, UVData
 import os
 import hera_cal
 import copy
+import ephem
```

Comment:
Wasn't there some reason we were looking to remove ephem as a dependency and use astropy instead? I vaguely remember hearing that. Maybe @plaplant knows?

Reply:
@plaplant says it checks out: once a Julian date is converted to UTC time via `astropy`, we can use `ephem` to get the zenith RA.

Reply:
Ok, cool.